content/60.methods.md

```diff
@@ -4,6 +4,19 @@
 
 ### Introduction
 
+As international attention remains focused on the ongoing public health crisis, the scientific community has responded by mobilizing resources and turning much of its attention to the virus and disease.
+This rapid influx of information is disseminated by traditional publishing mechanisms, preprint servers, and press releases, which provide a venue for scientists to release findings without undergoing the formal publication process.
+While having information available is valuable to efforts to understand and combat COVID-19, many contributions come from researchers across a wide range of fields who have varying degrees of experience working on coronaviruses and related topics.
+The volume of information available, much of which has not gone through rigorous peer review, presents a significant challenge to individual efforts to keep abreast of the state of COVID-19 research [@doi:10.1038/s42254-020-0175-7].
+However, research on these topics is proceeding so rapidly that any static review is likely to quickly become dated.
+Our goal as a community is to consolidate information about the virus in the context of related viruses and to synthesize rapidly emerging literature centered on the diagnosis and treatment of COVID-19.
+We used an open publishing framework, Manubot [@doi:10.1371/journal.pcbi.1007128], to manage hundreds of contributions from the community to create a living, scholarly document.
+We designed software to generate figures, such as Figure @fig:csse-deaths, that automatically update using external data sources.
```

Comment:
Is there code for this somewhere? Does it make sense to link to it here/add it to this repo?

Reply:
@ajlee21 It definitely should! That manuscript needs some major work. I'm tagging @mprobson who has said he'd like to put some work into making it read more like a computer/data science education paper this week!

Reply:
@ajlee21 this figure generation code is in this repo but on a separate branch.  You can find it in the subdirectories here https://github.com/greenelab/covid19-review/tree/external-resources

Linking to it directly once we start working on this standalone manuscript is a good idea.  I also tried archiving the entire repository with Software Heritage in the fall, but it failed.  We'll want a backup plan for archiving this code.